sql/partitioning.sql

```diff
@@ -0,0 +1,48 @@
+
+/* connect as migration user */
+\connect - migrator
+
+SET client_min_messages = WARNING;
+
+/* set up staging schemas */
+SELECT db_migrate_prepare(
+   plugin => 'ora_migrator',
+   server => 'oracle',
+   only_schemas => ARRAY['TESTSCHEMA3'],
+   options => JSONB '{"max_long": 1024}'
+);
+
+/* convert values based on Oracle function */
+UPDATE pgsql_stage.subpartitions
+   SET values = oracle_translate_expression(values)
+ WHERE schema = 'testschema3'
+   AND table_name = 'part4';
```

Comment:
I think the `db_migrator` part should do that automatically.
That's what the `translate_expression_fun` is there for.

Reply:
Call `oracle_translate_expression()` on values is included during array construct

https://github.com/fljdin/ora_migrator/commit/67f9a33c6e9e74a0d86f12a9bad6437def9a0a7c